### ISSUE TITLE
Some Screen-Text missing

### ISSUE THREAD

**Issue by sirsteve1** [Try/OpenGothic#351]
Dear @Try 

After updating to the latest master branch I noticed that during dialogs with NPCs when handing over or receiving items the screen text is now missing. This issue is new, as before the update the screen text was there. A random guess would be the update to pheonix, but I am not sure. What surprises me is that only specific screen texts are affected. Normal dialogs, the dialog text box, also receiving experience is not affected. Marvin messages do also work. For the moment only the missing item exchange notices are catching my eye.

Regards,
Steve

**Comment by lmichaelis** [comment]
Yeah, most new issues are probably related to the phoenix merge. I'll take a look later :)

**Comment by lmichaelis** [comment]
Well this is embarrassing ...

https://github.com/Try/OpenGothic/blob/fbd3c233a8445ab85d889db22904e04c8469e09b/game/game/gamescript.cpp#L2873-L2879

**Comment by Try** [comment]
Thanks @sirsteve1 and @lmichaelis !
Merged!
